javascript/javascript-5.md

```diff
@@ -49,17 +44,23 @@ Techtonica staff will assign pairs.
 
 Refer back to the calculator activity from the JavaScript 1 lesson. Refactor your code to replace your if/else statement(s) with switch statement(s).
 
-### Challenge
+### Challenges
 Refer back to the movie night activity from the JavaScript 3 lesson. Refactor your code to replace your if/else statement(s) with switch statement(s).
 
-### Extensions
```

Comment:
Actually, this is Extensions because it is optional.  can you keep the title but move these to after the Check For Understanding?

Reply:
Done